rust/otap-dataflow/xtask/src/main.rs

```diff
@@ -76,6 +78,20 @@ fn test_all() -> anyhow::Result<()> {
     Ok(())
 }
 
+fn compile_proto_otlp() -> anyhow::Result<()> {
+    tonic_build::configure()
+        .out_dir("crates/otlp/src/grpc_stubs")
+        .compile_protos(
+            &[
+                "crates/otlp/proto/opentelemetry/proto/collector/logs/v1/logs_service.proto",
+                "crates/otlp/proto/opentelemetry/proto/collector/metrics/v1/metrics_service.proto",
+                "crates/otlp/proto/opentelemetry/proto/collector/trace/v1/trace_service.proto",
+            ],
+            &["crates/otlp/proto"],
+        )?;
+    Ok(())
+}
```

Comment:
Sorry for my confusion above. I think it's good to check-in the generated Rust files created in `build.rs`, but we already have a submodule for the protocol, and all it takes is to change this stuff.

`otel-arrow-rust/build.rs` uses an expression like:

            &[format!("{}/../../proto/opentelemetry/proto/", base)],

Will you please do a similar thing here, restore the submodule and avoid checking in the .proto files?